src/otpch.h

```diff
@@ -7,24 +7,49 @@
 // Definitions should be global.
 #include "definitions.h"
 
+// System headers required in headers should be included here.
 #include <algorithm>
+#include <array>
+#include <atomic>
+#include <bitset>
 #include <boost/asio.hpp>
+#include <boost/iostreams/device/mapped_file.hpp>
+#include <boost/lockfree/stack.hpp>
+#include <boost/variant.hpp>
+#include <cassert>
+#include <condition_variable>
+#include <cryptopp/rsa.h>
 #include <cstdint>
+#include <cstdlib>
+#include <deque>
 #include <fmt/color.h>
 #include <forward_list>
 #include <functional>
 #include <iostream>
+#include <limits>
 #include <list>
 #include <map>
 #include <memory>
 #include <mutex>
+#include <mysql/mysql.h>
 #include <pugixml.hpp>
+#include <random>
 #include <set>
 #include <sstream>
+#include <string_view>
 #include <string>
 #include <thread>
 #include <unordered_map>
 #include <unordered_set>
+#include <utility>
+#include <valarray>
+#include <variant>
 #include <vector>
 
+#if __has_include("luajit/lua.hpp")
```

Comment:
didn't we drop this __has_include thing for some reason?

Reply:
I don't remember exactly, but it could have something to do with the CMAKE's `-DUSE_LUAJIT` not working right.